### PR TITLE
shallow copies are evil. Copy the DOM node properly.

### PR DESCRIPTION
**Pull request by benadida**
addresses #114.

### CODE REVIEW

File: client/src/components/HandMarkedPaperBallot.tsx [hunk 1]
```diff
@@ -133,28 +133,39 @@ class PostRenderBallotProcessor extends Handler {
...
+        newLastPageElement.classList.remove(
+          'pagedjs_first_page',
+          'pagedjs_right_page'
+        )
+        newLastPageElement.classList.add('pagedjs_left_page')
+        newLastPageElement.setAttribute(
+          'data-page-number',
+          `${pages.length + 1}`
+        )
+        newLastPageElement.setAttribute('data-id', `page-${pages.length + 1}`)
```
Comment (eventualbuddha) [inline client/src/components/HandMarkedPaperBallot.tsx:154]: Is there a reason not to use the approach we had, i.e.

```suggestion
        newLastPageElement.dataset.pageNumber = `${pages.length + 1}`
        newLastPageElement.dataset.id = `page-${pages.length + 1}`
```

File: client/src/components/HandMarkedPaperBallot.tsx [hunk 2]
```diff
@@ -133,28 +133,39 @@ class PostRenderBallotProcessor extends Handler {
...
+          newLastPageElement.getElementsByClassName('pagedjs_page_content')[0]
         )
 
-        const newPage = { ...pages[pages.length - 1] }
-        newPage.element.dataset.pageNumber = `${pages.length + 1}`
-        newPage.element.dataset.id = `page-${pages.length + 1}`
+        const newPage = {
+          ...pages[pages.length - 1],
+          element: newLastPageElement,
+        }
```
Comment (eventualbuddha) [inline client/src/components/HandMarkedPaperBallot.tsx:167]: This is the actual fix, right? Everything else should be equivalent.

**Review by eventualbuddha — APPROVED** [review]

File: client/src/components/HandMarkedPaperBallot.tsx (hunk 1 above)
Comment (benadida) [reply client/src/components/HandMarkedPaperBallot.tsx:154]: maybe, but the approach we had was via `setAttribute` so I didn't want to change it. I added the `id` one cause it was missing.

File: client/src/components/HandMarkedPaperBallot.tsx (hunk 2 above)
Comment (benadida) [reply client/src/components/HandMarkedPaperBallot.tsx:167]: should, yes. Trying to change as little as possible.

**Pull request merged**